client/src/pages/ProfileDetails/BookingForm/BookingForm.tsx

```diff
@@ -0,0 +1,194 @@
+import useStyles from './useStyles';
+import { Box, Typography, Select, MenuItem, OutlinedInput, Grid, Button, Card } from '@material-ui/core';
+import Rating from '@material-ui/lab/Rating';
+import { useFormik } from 'formik';
+import fetchProfile from '../../../helpers/APICalls/fetchProfile';
+import { useEffect, useState } from 'react';
+import createRequestNotification from '../../../helpers/APICalls/createRequestNotification';
+import { mockName } from '../../../mocks/mockId';
+
+const times = [...Array(24).keys()].map((number) => {
+  if (number === 0) {
+    return {
+      time: '12:00 AM',
+      value: number,
+    };
+  } else if (number < 12) {
+    return {
+      time: number.toString() + ':00 AM',
+      value: number,
+    };
+  } else if (number === 12) {
+    return {
+      time: number.toString() + ':00 PM',
+      value: number,
+    };
+  } else {
+    return {
+      time: (number - 12).toString() + ':00 PM',
+      value: number,
+    };
+  }
+});
+
+const BookingForm = (props: { id: string }): JSX.Element => {
+  const classes = useStyles();
+  const [availableTime, setAvailableTime] = useState([
+    {
+      day: 'Monday-default',
+      startTime: '8:00 AM - default',
+      endTime: '10:00 PM - default',
+    },
+  ]);
+  const [availableStartHours, setAvailableStartHours] = useState([
+    {
+      time: '12:00 AM - default',
+      value: 0,
+    },
+  ]);
+  const [availableEndHours, setAvailableEndHours] = useState([
+    {
+      time: '11:00 PM - default',
+      value: 23,
+    },
+  ]);
+
+  useEffect(() => {
+    fetchProfile(props.id).then((data) => setAvailableTime(data.success.profile.availableTime));
+  }, [props.id]);
+
+  const formik = useFormik({
+    initialValues: {
+      startDate: 'placeholder',
+      startTime: 'placeholder',
+      endDate: 'placeholder',
+      endTime: 'placeholder',
+    },
+    onSubmit: (values) => {
+      alert('Your request for service has been submitted!');
```

Comment:
It's nice to see some form of feedback being communicated with the user, although I suggest we do this through [Snackbars](https://v4.mui.com/components/snackbars/#snackbar) instead (we already have the context implemented in the codebase ([here](https://github.com/hatchways/team-gingerbread/blob/justin-integration-profile-details/client/src/context/useSnackbarContext.tsx)) 🎉) !

Reply:
@jb162 Please consider using the Snackbar here 👍🏽